lib/intercom_stats/intercom/worker.ex

```diff
@@ -115,21 +115,14 @@ defmodule IntercomStats.Intercom.Worker do
     conversation = request_conversation(item)
     response_times = calculate_response_times(conversation)
     closed_timestamp = determine_closed_timestamp(conversation)
-    snooze_time = determine_snooze_time(conversation)
+    closing_time = determine_open_time(conversation)
```

Comment:
it seems odd that the `closing_time` is given by a function called `determine_open_time`, what does `closing_time` describe?